docs/File-System-API.md

```diff
@@ -58,14 +58,69 @@ OutStreamOptions options = new OutStreamOptions.Builder(ClientContext.getConf())
 FileOutStream out = tfs.getOutputStream(path, options);
 ```
 
+### IO Options
+
+There are two different types of storage in Tachyon, Tachyon managed storage and under storage.
+Tachyon managed storage is the memory, SSD, and/or HDD which is allocated to Tachyon workers. For
+read and write operations, users can specify the `TachyonStorageType` to fine tune the interaction
+with Tachyon's managed storage. Likewise, users can specify the `UnderStorageType` to fine tune the
+data operations with the under storage, such as S3, Swift, or HDFS. Note that read operations are
+affected by `TachyonStorageType` (the data can be stored in more Tachyon workers) but not
+`UnderStorageType`.
+
+Below is a table of commonly used `TachyonStorageType` and `UnderStorageType` combinations.
+
+<table class="table">
```

Comment:
Do `TachyonStorageType` and `UnderStorageType` interact with each other at all in this table? It seems like just defining `PROMOTE`, `STORE`, `NO_STORE`, `SYNC_PERSIST`, and `NO_PERSIST` would be a simpler way to convey the same information